apps/Directory.mk

```diff
@@ -64,7 +64,7 @@ all: nothing
 
 define SDIR_template
 $(1)_$(2):
-	$(Q) $(MAKE) -C $(1) $(2) TOPDIR="$(TOPDIR)" APPDIR="$(APPDIR)"
+	$(Q) $(MAKE) -C $(1) $(2) TOPDIR="$(TOPDIR)" APPDIR="$(APPDIR)" EXTDIR="$(EXTDIR)"
```

Comment:
How do you get EXTDIR? When makefile of apps is called, EXTDIR is not passed by makefile.unix.